buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -474,6 +474,9 @@ def get_pipeline_name(spec, version) {
 */
 def get_summary_table(identifier) {
     // fetch the downstream builds of the current build
+    if (!env.buildFile) {
```

Comment:
buildFile is not an environment variable, it is a global variable, replace with `buildFile`

Reply:
Yea this doesn't work. I don't think `buildFile` by itself will work either as the variable will be undefined. I think my original solution is our best option.

Reply:
buildFile is defined on top after the checkout https://github.com/eclipse/openj9/blob/652888eb1aca2a02f4f4ac124da073ce9cc3f76c/buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy#L241
Why this check here? It would fail earlier if buildFile cannot be loaded.

Reply:
Right, but if the failure is at or before the load, then when this runs in the finally it throws an error because buildFile is undefined. But the error is misleading because it's not what caused the build to fial.

Reply:
Perhaps the better answer is to skip it when it is undefined rather than throwing an error. Not throwing will allow the initial error to be printed.